referenceframe/frame_system.go

```diff
@@ -371,6 +359,56 @@ func (sfs *simpleFrameSystem) composeTransforms(frame Frame, inputMap map[string
 	return q, errAll
 }
 
+// MergeFrameSystems will merge fromFS into toFS with an offset frame given by cfg. If cfg is nil, fromFS
```

Comment:
How is this function related to the PR? Is this a utility we want to provide for working with remote geometries?

Reply:
This function was previously in `config/frame.go`, which has been deleted as part of this work, as everything inside it is duplicated within `referenceframe`.

Since this function operates entirely on `referenceframe` structs, it makes sense to put it here rather than moving it elsewhere inside `config`

Reply:
Thanks!